cmd/getbalance.go

```diff
@@ -14,6 +15,8 @@ var getbalancecmd = &cobra.Command{
 	Long:  `Get balance from sharders`,
 	Args:  cobra.MinimumNArgs(0),
 	Run: func(cmd *cobra.Command, args []string) {
+		doJSON, _ := cmd.Flags().GetBool("json")
```

Comment:
Check error in case use invalid value is used

Reply:
--json is bool so requires no parameter. But feel free to change however is best way

Reply:
@peterlimg do you agree? there is no parameter to check?